src/cli/exceptions.py

```diff
@@ -1,39 +1,98 @@
+from pathlib import Path
```

Comment:
It looks like you have created an exception for every single validation that can fail. That is not what we want when checking validity of the application, because we want to print all validation errors we find.
We only raise an exception in the case of an unrecoverable error we agreed last week.